multigeiger/speaker.cpp

```diff
@@ -22,6 +22,15 @@ volatile int *isr_tick_sequence = NULL;
 volatile int *isr_sequence = NULL;  // currently played sequence
 
 static int tick_sequence[8];
+static int alarm_sequence[12] = {
+  // "high_Pitch"
+  3000000, 1, -1, 400,  // frequency_mHz, volume, LED, -1 = don't touch, duration_ms
```

Comment:
maybe rather:

// frequency_mHz, volume, LED (-1 = don't touch), duration_ms

if this is badly phrased at other places also, this could be a separate cleanup PR.

Reply:
guess it was just a bad phrasing by me, corrected